src/Ninja/G1CP/Content_G1.src

```diff
@@ -32,7 +32,8 @@ Content\Misc\npc.d
 
 // Fixes and tests
 Content\Fixes\record.d
-Content\testsuite.d
+Content\Testsuite\testsuite.d
+Content\Testsuite\getter.d
```

Comment:
Since not all of the functions are getter functions now, should change the name of the file to something like `checks.d` or `conditions.d`?